admin-dev/themes/new-theme/package.json

```diff
@@ -4,47 +4,55 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "postinstall": "webpack --progress",
-    "build": "webpack --progress --colors --debug --display-chunks",
-    "watch": "webpack --progress --colors --debug --display-chunks --watch"
+    "postinstall": "webpack",
+    "start-dev-server": "webpack-dev-server --config webpack.config.js"
   },
   "author": "",
   "license": "ISC",
   "devDependencies": {
     "PrestaKit": "git+https://github.com/PrestaShop/prestashop-ui-kit.git",
     "babel-core": "^6.4.5",
     "babel-loader": "^6.2.1",
+    "babel-preset-env": "^1.2.1",
     "babel-preset-es2015": "^6.3.13",
+    "babel-preset-stage-2": "^6.22.0",
     "bootstrap": "4.0.0-alpha.2",
     "bootstrap-slider": "^6.0.16",
     "bootstrap-tokenfield": "^0.12.0",
     "bourbon": "^4.2.6",
     "compass": "^0.1.1",
     "css-loader": "^0.23.1",
     "dropzone": "^4.2.0",
-    "eonasdan-bootstrap-datetimepicker": "^4.15.35",
+    "eonasdan-bootstrap-datetimepicker": "^4.17.47",
     "exports-loader": "^0.6.3",
-    "expose-loader": "^0.7.1",
-    "extract-text-webpack-plugin": "^1.0.1",
-    "jets": "0.11.0",
-    "file-loader": "^0.8.5",
+    "expose-loader": "^0.7.3",
+    "extract-text-webpack-plugin": "^2.1.0",
+    "file-loader": "^0.10.1",
     "imports-loader": "^0.6.5",
+    "jets": "0.11.0",
     "jquery": "^2.2.0",
     "jquery-ui": "1.10.5",
     "jwerty": "^0.3.2",
+    "loader-utils": "^1.0.3",
+    "lodash": "^4.17.4",
     "magnific-popup": "^1.0.1",
-    "moment": "^2.11.2",
-    "node-sass": "^3.7",
+    "mobile-detect": "^1.3.5",
+    "node-sass": "^3.13.1",
     "path": "^0.12.7",
+    "postcss-cssnext": "^2.10.0",
     "postcss-loader": "^0.8.0",
     "sass-loader": "^3.1.2",
     "sprintf-js": "^1.0.3",
     "style-loader": "^0.13.0",
     "tether": "^1.3.2",
     "typeahead.js": "^0.11.1",
-    "webpack": "^1.12.12"
-  },
-  "dependencies": {
-    "mobile-detect": "^1.3.5"
+    "vue": "^2.1.10",
```

Comment:
@xBorderie it's like: "damn, i left and they started to use cool stuff" :D i hope!